pkg/syncer/fullsync.go

```diff
@@ -116,6 +116,65 @@ func CsiFullSync(ctx context.Context, metadataSyncer *metadataSyncInformer) erro
 		return err
 	}
 
+	if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorWorkload &&
```

Comment:
We cannot merge this PR till CNS gets back on how to remove the clusterID from ContainerClusterArray right?

Reply:
We can merge CSI side changes as it is guarded with the `tkgs-ha` feature gate.
we need to use the new `supervisor-id` so this change is required.